### PR TITLE
Update CHF translation (#144)

### PR DESCRIPTION
**Pull request by emirhanbalkac**

### CODE REVIEW

**Comment by emirhanbalkac** [comment]
Hi @kwn. Can you review it?

File: src/Legacy/Numbers/Words/Locale/Tr.php [hunk 1]
```diff
@@ -76,7 +76,7 @@ class Tr extends Words
         'BWP' => [['Botswana pulası'], ['thebe']],
         'BYR' => [['Belarus rublesi'], ['kopiejka']],
         'CAD' => [['Kanada doları'], ['sent']],
-        'CHF' => [['İsveç frankı'], ['rapp']],
+        'CHF' => [['İsviçre frankı'], ['rapp']],
```
Comment (uguurozkan) [inline src/Legacy/Numbers/Words/Locale/Tr.php:79]:
```suggestion
        'CHF' => [['İsviçre frangı'], ['rapp']],
```

It's written as `İSVİÇRE FRANGI` in [the Central Bank of the Republic of Türkiye's website](https://www.tcmb.gov.tr/kurlar/today.xml#:~:text=%C4%B0SV%C4%B0%C3%87RE%20FRANGI).

Comment (uguurozkan) [inline src/Legacy/Numbers/Words/Locale/Tr.php:79]:
```suggestion
        'CHF' => [['İsviçre frangı'], ['rapp']],
```

It's written as `İSVİÇRE FRANGI` in [the Central Bank of the Republic of Türkiye's website](https://www.tcmb.gov.tr/kurlar/today.xml#:~:text=%C4%B0SV%C4%B0%C3%87RE%20FRANGI).

**Review by uguurozkan — CHANGES_REQUESTED** [review]

File: src/Legacy/Numbers/Words/Locale/Tr.php (hunk 1 above)
Comment (emirhanbalkac) [reply src/Legacy/Numbers/Words/Locale/Tr.php]: > It's written as `İSVİÇRE FRANGI` in [the Central Bank of the Republic of Türkiye's website](https://www.tcmb.gov.tr/kurlar/today.xml#:~:text=%C4%B0SV%C4%B0%C3%87RE%20FRANGI).

Thanks :)

**Review by kwn — CHANGES_REQUESTED** [review]
Could you please add a few unit tests for various amounts?

**Comment by emirhanbalkac** [comment]
> Could you please add a few unit tests for various amounts?

Done. Can you check it?

**Review by uguurozkan — APPROVED** [review]

**Review by kwn — APPROVED** [review]

**Comment by kwn** [comment]
Thank you!

**Pull request merged**